src/ImageGallery.jsx

```diff
@@ -116,6 +117,7 @@ export default class ImageGallery extends React.Component {
     swipingTransitionDuration: 0,
     slideInterval: 3000,
     swipeThreshold: 30,
+    preLoad: 0,
```

Comment:
What about `numberImagesPreLoad`? Would like something a little more descriptive

Reply:
If we're going verbose, why not go all in?

`numberOfImagesToPreLoad`